main.go

```diff
@@ -336,10 +307,12 @@ func resolvePaths(paths, skip []string) []string {
 					return filepath.SkipDir
 				case !i.IsDir() && !skip && strings.HasSuffix(p, ".go"):
 					dirs.add(filepath.Clean(filepath.Dir(p)))
+					// dirs.add(filepath.Clean(p)) // DISCARD Make it allow file paths too. see below
```

Comment:
Why do we need filepaths?

Reply:
I have removed the comment here. Support was found for filepaths below that block of code .